src/components/transactions/message-view/MultiSendMessageDetails.vue

```diff
@@ -60,12 +55,35 @@ export default {
     }
   },
   computed: {
+    recipient() {
```

Comment:
what are you doing here? why the conditional that the user address is in the inputs?